### PR TITLE
SDK-332: Correct the O3 Versions displayed during setup of an instance

### PR DESCRIPTION
**Pull request by mherman22**
## Issue I worked on

see https://openmrs.atlassian.net/browse/SDK-332

## screenshot(s)
Screenshot from 2024-08-20 11-41-09

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean install` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute the above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/VersionsHelper.java [hunk 1]
```diff
@@ -68,14 +69,14 @@ private List<ArtifactVersion> getVersions(Artifact artifact){
 
     public List<ArtifactVersion> getAllVersions(Artifact artifact, int maxSize) {
         List<ArtifactVersion> versions = getVersions(artifact);
+        versions.removeIf(version -> "3.1.0-SNAPSHOT".equals(version.toString()));
+        versions.removeIf(version -> "3.0.0-SNAPSHOT".equals(version.toString()));
```
Comment (ibacher) [inline sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/VersionsHelper.java:73]: Ooh... I don't exactly like doing this because we'd need to constantly maintain this. Is there a way we can write some logic that would

[the remark this answers is not in the document]
Comment (mherman22) [reply sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/VersionsHelper.java:73]: Makes sense, lemi make the change

[the remark this answers is not in the document]
Comment (mherman22) [reply sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/VersionsHelper.java]: done!

File: sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java [hunk 2]
```diff
@@ -782,11 +782,11 @@ private Map<String, String> getDistroVersionsOptionsMap(Set<String> versions, Ve
 	 * @return A LinkedHashMap containing the generated options map.
 	 */
 	private Map<String, String> getO3VersionsOptionsMap(VersionsHelper versionsHelper,
-			String optionTemplate) {
+														String optionTemplate) {
```
Comment (ibacher) [inline sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java:785]: Could we undo the change to this line?

**Review by ibacher — APPROVED** [review]

File: sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java (hunk 2 above)
Comment (mherman22) [reply sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java:785]: next commit?
